go/logictest/runner.go

```diff
@@ -433,8 +434,9 @@ func logSuccess() {
 }
 
 func logMessagePrefix() string {
-	return fmt.Sprintf("%s %s:%d: %s",
+	return fmt.Sprintf("%s %d %s:%d: %s",
 		time.Now().Format(time.RFC3339Nano),
+		time.Now().Sub(startTime).Nanoseconds(),
```

Comment:
I would go with milliseconds here. Nanos are unlikely to be meaningful for timing purposes.

Reply:
The only reason we're using nanos for the timestamp is that there isn't a predefined format with millis.